### ISSUE TITLE
OpenML doesn't return deactivated datasets any more by calling data/list

### ISSUE THREAD

**Issue by mfeurer** [openml/openml-python#149]
This makes a test fail which checks whether dataset 17 is deactivated. Right now, it is not documented on OpenML how to retrieve deactivated datasets. Can only fix this issue once the documentation on OpenML is done.

**Comment by joaquinvanschoren** [comment]
You mean this?
https://www.openml.org/api/v1/xml/data/list/status/deactivated

**Comment by joaquinvanschoren** [comment]
Also documented:
https://www.openml.org/api_docs#!/data/get_data_list_status_status

I now notice how old this issue is. I just noticed because one of the unit tests is skipped because of it :)

**Comment by mfeurer** [comment]
Yeah, we need to update the package at some point to reflect this change - I'm leaving this open for this reason.

**Comment by PGijsbers** [comment]
Is it possible this issue is outdated? For deactivated dataset 478, I can currently fetch and discover it thus (all work):
```
d = openml.datasets.get_dataset(478)

ds = openml.datasets.list_datasets(status='all')
ds[478]  # No key error, description present

ds = openml.datasets.list_datasets(status='deactivated')
ds[478]  # No key error, description present
```
Deactivated datasets are by default *not* returned for `list_datasets`, but this is documented in the function's docstring.

**Comment by PGijsbers** [comment]
Nevermind I noticed the unit test referenced in #382. Though I do think it is a separate issue (I think it is `check_datasets_active` that has the bug).

**Comment by mfeurer** [comment]
Fixed via #642.
